src/ucp/core/ucp_context.c

```diff
@@ -1031,6 +1031,13 @@ static void ucp_fill_sockaddr_prio_list(ucp_context_h context,
                                     num_sockaddr_tls);
     ucp_fill_sockaddr_cms_prio_list(context, sockaddr_tl_names,
                                     num_sockaddr_tls, sockaddr_cm_enable);
+    if ((context->config.ext.sockaddr_cm_enable == UCS_YES) &&
+        (context->config.num_cm_cmpts == 0)) {
+        ucs_error("UCX_SOCKADDR_CM_ENABLE is set to yes but no one component supports CM");
```

Comment:
"UCX_SOCKADDR_CM_ENABLE is set to yes but none of the available components supports CM"
can we print the available components to the user to make this message more informative?

Reply:
do you think list of components is useful even if none of them supports required capability?

Reply:
you say in the message that none of the components supports CM. do you think the user knows which components we tested for it?

Reply:
He is trying to enable UCX_SOCKADDR_CM_ENABLE explicitly, IMO message like his system does not support this feature is enough.

Reply:
this value (yes) will become the default soon so it won't be set explicitly. 
IMO it's better to provide more useful information when it's possible, not only for the user but also for us when we will be debugging.

Reply:
> this value (yes) will become the default soon

Should not "try" to be set by default?
> to provide more useful information when it's possible

list of present components is useless, do you think we have to point to required SW & HW? ~~link to the market?~~ :)